src/composables/multisig.ts

```diff
@@ -13,13 +20,19 @@ interface UseMultisigOptions {
   store: Store<any>
 }
 
+const POLLING_INTERVAL = 5000;
+
 /**
  * This composable provides methods related to multisig operations
  */
 export function useMultisig({ store }: UseMultisigOptions) {
   const { getDrySdk, getSdk } = useSdk({ store });
+  const activeNetwork = computed<INetwork>(() => store.getters.activeNetwork);
 
   const multisigProgress = ref<IMultisigCreationStep>(MULTISIG_CREATION_STEPS.preparing);
+  const activeMultisigTransactions = ref<Record<string, IActiveMultisigTx>>({});
```

Comment:
A  map where the key is the multisig account Id and the value is the active transaction with all the consensus statuses. This transaction detail could be merged with the unified multisig account data structure too.